boards/arm/fvp_base_revc_2xaemv8a/fvp_base_revc_2xaemv8a_fvp_base_revc_2xaemv8a_smp_ns.yaml

```diff
@@ -1,7 +1,7 @@
 # Copyright (c) 2022 Arm Limited (or its affiliates). All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-identifier: fvp_base_revc_2xaemv8a//smp/ns
```

Comment:
Why is this changed?

As far as I understand omitting the SoC when there's only one is permitted:

from https://docs.zephyrproject.org/latest/hardware/porting/board_porting.html#board-terminology

> If a board contains only one single-core SoC, then the SoC can be omitted from the board target. This implies that if the board does not define any board qualifiers, the board name can be used as a board target. Conversely, if board qualifiers are part of the board definition, then the SoC can be omitted by leaving it out but including the corresponding forward-slashes: //.